meson.build

```diff
@@ -0,0 +1,18 @@
+project('sdl2_ttf', 'c',
+    version : '2.0.12',
+    license : 'zlib',
+    subproject_dir : 'external',
+    default_options : ['c_std=gnu99'])
+
+freetype_dep = subproject('freetype2', required: true)
+sdl2_dep = dependency('sdl2', fallback : ['sdl2', 'sdl2_dep'], required: true)
+
```

Comment:
Unfortunately the build failed with
```
[44/44] Linking target libsdl2_ttf.so
FAILED: libsdl2_ttf.so
cc  -o libsdl2_ttf.so libsdl2_ttf.so.p/SDL_ttf.c.o -Wl,--as-needed -Wl,--no-undefined -shared -fPIC -Wl,--start-group -Wl,-soname,libsdl2_ttf.so external/freetype-2.9.1/libfreetype.so.6.16.1 /usr/lib/libharfbuzz.so /usr/lib/libz.so /usr/lib/libpng16.so -pthread /usr/lib/libSDL2.so -Wl,--end-group '-Wl,-rpath,$ORIGIN/external/freetype-2.9.1' -Wl,-rpath-link,/tmp/tmp4j4srqvn/bin/external/freetype-2.9.1
/usr/bin/ld: libsdl2_ttf.so.p/SDL_ttf.c.o: in function `Load_Glyph':
/tmp/tmp4j4srqvn/bin/../src/SDL2_ttf-2.0.12/SDL_ttf.c:649: undefined reference to `ceil'
/usr/bin/ld: /tmp/tmp4j4srqvn/bin/../src/SDL2_ttf-2.0.12/SDL_ttf.c:733: undefined reference to `ceil'
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
ninja build: ☒
```

I think a fix can be to add `-lm`: https://mesonbuild.com/howtox.html#add-math-library-lm-portably
```
cc = meson.get_compiler('c')
m_dep = cc.find_library('m', required : false)

sdl2_ttf = library(... dependencies : [..., m_dep])
```

Reply:
Ok added quick fix hopefully it will work this time.